packages/babel-plugin-transform-imports/lib/index.js

```
@@ -62,61 +62,52 @@ function transform(transformOption, importName, matches) {
-        const sourceRoot = dirname(sourceIndex)
-
-        const importPaths = globby.sync(
-          [
-            `${sourceRoot}/**/${importName}.js`,
-            `${sourceRoot}/**/${importName}/index.js`
-          ],
-          { cwd: sourceRoot }
-        )
-
-        if (!importPaths || importPaths.length === 0) {
-          // If there are no import paths found it is the same as if globby or the require.resolve failed, we cannot construct a relative import path
-          // so throw an error and just fall back to using the import name as the path.
-          throw Error()
-        }
-
-        if (importPaths.length > 1) {
-          console.error(
-            `[${transformName}] multiple modules found with the name '${importName}' in '${packageName}'. Continuing using the first match: '${importPaths[0]}'.`
-          )
-        }
-
-        importPath = relative(
-          sourceRoot,
-          importPaths[0].endsWith('index.js')
-            ? dirname(importPaths[0])
-            : importPaths[0]
-        )
-      } catch (error) {
+      if (!importPaths || importPaths.length === 0) {
+        // If there are no import paths found it is the same as if globby or the require.resolve failed, we cannot construct a relative import path
+        // so throw an error and just fall back to using the import name as the path.
         console.error(
           `[${transformName}] no modules match '${importName}' in '${packageName}'. Continuing with '${importName}' as the import path. If that is unexpected or incorrect, make sure '${importName}' exists in '${packageName}' and you have run 'yarn install' to download the package source.`
         )
+        return null
       }
+      if (importPaths.length > 1) {
+        console.error(
+          `[${transformName}] multiple modules found with the name '${importName}' in '${packageName}'. Continuing using the first match: '${importPaths[0]}'.`
+        )
```

This is also a weird edge case, should we throw an error in this case?